Sources/TypeChecker/TypeChecker+Expressions.swift

```diff
@@ -173,6 +173,20 @@ extension TypeChecker {
     return ASTPassResult(element: functionCall, diagnostics: diagnostics, passContext: passContext)
   }
 
+  public func process(externalCall: ExternalCall, passContext: ASTPassContext) -> ASTPassResult<ExternalCall> {
```

Comment:
Are you sure this should be in the TypeChecker?

Reply:
Moved to SemanticAnalyzer+Expressions